packages/types/src/primitive/Extrinsic/v2/Extrinsic.spec.ts

```diff
@@ -5,19 +5,20 @@
 import '../../../injector';
 
 import BN from 'bn.js';
-import extrinsics from '@polkadot/api-metadata/extrinsics/static';
+import Decorated from '@polkadot/api-metadata/Decorated';
+import Metadata from '@polkadot/api-metadata/Metadata';
```

Comment:
Same comment as above, all the files between the previous comment and this one - we should really just work with the wrapper.